### PR TITLE
Release 2.1.0

### PR DESCRIPTION
**Pull request by dead-horse**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+
+* **feature**
+  * add 400 response for broken client request to instead of empty response
+  * dump application router json
+
+* **fix**
+  * fix: run dumpConfig at the last ready callback
+
+* **document**
+  * migrate docs to egg 2
```
Comment (atian25) [inline History.md]: 还有几个：
* docs(passport): the missing docs for passport (#1824)

File: History.md [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+
+### Commits
+
+* [[`40df153`](http://github.com/eggjs/egg/commit/40df153dd7ca8124a7502ba6cdc838835388a0ae)] - feat: add 400 response for broken client request to instead of empty response (#1829) (Khaidi Chu <<i@2333.moe>>)
```
Comment (fengmk2) [inline History.md]: @XadillaX 下次 commit log 一行不要超长。

Comment (XadillaX) [reply History.md]: 感觉这个是被自动 merge 给坑了 -。 -

**Comment by dead-horse** [comment]
2.1.0 released

**Pull request merged**